### ISSUE TITLE
Function selector arguments should be on the same line

### ISSUE THREAD

**Issue by k06a** [duaraghav8/Ethlint#257]
**Description**
Following solidity 0.5.x code:
```Solidity
bytes4 private constant InterfaceId_ERC721Countable = (
    this.totalAmount().selector ^
    this.tokenAmount(uint256).selector ^
    this.approveAmount(address,uint256,uint256).selector ^
    this.transferFrom(address,address,uint256,uint256).selector ^
    this.safeTransferFrom(address,address,uint256,uint256).selector ^
    this.safeTransferFrom(address,address,uint256,uint256,bytes).selector
);
```

Gives 3 warnings:
```
Function "undefined": in case of more than 3 arguments, drop each into its own line.    arg-overflow
```

**Linter version**
1.2.2

**Priority**
`Medium`

**Comment by duaraghav8** [comment]
@k06a Can you paste your full solidity code and `.soliumrc.json`? I think I know why these warnings show up but I need to see the setup to verify

**Comment by k06a** [comment]
@duaraghav8 sorry, it seems this code it not yet valid for solidity 0.5.3

**Comment by duaraghav8** [comment]
ah. But are you saying this code will be valid in a future release? (I haven't read solidity's roadmap, but I can track it to provide support when its finally introduced)

**Comment by k06a** [comment]
>But are you saying this code will be valid in a future release?

Hope so, but can't find it in roadmap
